css/style.css

```diff
@@ -0,0 +1,255 @@
+@font-face {
+  font-family: "Cocogoose";
+  src:
+    local("Cocogoose"),
+    url("../fonts/Cocogoose_trial.otf") format("opentype");
+}
+
+:root {
+  --primary-color: #ec5242;
+  --secondary-color: #272a31;
+  --tertiary-color: #d3d3d3;
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  border: 0;
+  vertical-align: baseline;
+}
+
+html {
+  font-size: calc(15px + 0.390625vw);
+}
+
+body {
+  font-family: "Lato", sans-serif;
+  overflow-x: hidden;
+}
+
+.logo {
+  width: 150px;
+  height: 50px;
+  margin-top: 10px;
+}
+
+.line-primary {
+  background-color: var(--primary-color);
+  height: 3px;
+  width: 100px;
+  display: inline-block;
+}
+
+.btn-danger {
+  background-color: var(--primary-color);
+  border-color: var(--primary-color);
+}
+
+.btn-danger:hover,
+.btn-danger:focus,
+.btn-danger:active {
+  background-color: var(--primary-color);
+  border-color: var(--primary-color);
+  filter: contrast(200%);
+}
+
+.header {
+  position: relative;
+  background: rgb(255, 255, 255);
+  background:
+    linear-gradient(
+      180deg,
+      rgba(255, 255, 255, 0) 63%,
+      rgba(39, 42, 49, 1) 100%
+    );
+}
+
+/* navbar */
+.nav-top .nav-link {
+  color: var(--secondary-color);
+}
+
+.navbar-toggler-icon {
+  background-image: none;
+}
+
+.navbar-toggler-icon::before {
+  font-family: FontAwesome, sans-serif;
+  content: "\f0c9";
+  color: var(--tertiary-color);
+  font-size: 30px;
+}
+
+/* change the link color */
+.navbar-custom .navbar-nav .nav-link {
+  color: var(--tertiary-color);
+  text-transform: uppercase;
+  cursor: pointer;
+}
+
+/* change the color of active or hovered links */
+.navbar-custom .nav-item.active .nav-link,
+.navbar-custom .nav-item:hover .nav-link {
+  color: #fff;
+}
+
+.nav-top .nav-item:hover .nav-link {
+  color: var(--primary-color);
+}
+
+/* headline */
+.headline {
+  background-color: var(--secondary-color);
+  background-image: url(../img/dex-ezekiel-IxDPZ-AHfoI-unsplash.jpg);
+  background-repeat: no-repeat;
+  background-size: contain;
+  position: absolute;
+  top: 0;
+  left: 0;
+  filter: blur(3px);
+}
+
+@media screen and (min-width: 768px) {
+  .headline {
+    background-size: cover;
+  }
+}
```

Comment:
Kindly ensure that the media queries are working properly. I can see that the mobile nav menu still lingers at certain desktop screen widths while the dark bottom bar dissapears as shown below. 
This could be because you have closed the media query on line 117.


About page

![Screenshot from 2023-01-12 08-52-40](https://user-images.githubusercontent.com/65233860/212022355-c33939a0-81dc-4312-b040-042fbed6b3c9.png)

Home page

![Screenshot from 2023-01-12 08-52-28](https://user-images.githubusercontent.com/65233860/212022314-d43cca18-051d-4d48-8ab1-b74f9588179c.png)